### ISSUE TITLE
LC0005 False Positives of casing

### ISSUE THREAD

**Issue by jwikman** [StefanMaron/BusinessCentral.LinterCop#90]
The LC0005 rule is really great! It helped me to find a lot of casing issues.

But the last update introduced a few false positives.

I at least found three of them (maybe there are more):
- `Action` datatype
- `Interface `datatype
- `Page `datatype

All those datatypes should begin with a capital letter, but LC0005 now thinks it should be all lowercase.

The following repro gets three warnings:
```
    procedure MyProcedure(MyAction: Action)
    var
        MyInterface: Interface "Price Asset";
        CustomerCard: Page "Customer Card";
    begin
    end;
```

The common thing with those keywords is that they all are both data types and another keyword. Don't know if that's the cause, though...
- `action` is used in pages.
- `interface` and `page` is object types.

**Comment by StefanMaron** [comment]
Thanks for reporting! 

I already managed to find `Codeunit`, `Enum`, and `Label`. But did not check the others

**Comment by StefanMaron** [comment]
Fix is released. Please report back if the false positives are gone

**Comment by rvanbekkum** [comment]
Was about to report the same. It also gives a false positive for `Report`.

**Comment by jwikman** [comment]
I assume that this will happen for all object types?

**Comment by jwikman** [comment]
The warnings I had earlier today is now gone. 👍

**Comment by StefanMaron** [comment]
> Was about to report the same. It also gives a false positive for `Report`.

Heck, I will fix it right away

**Comment by StefanMaron** [comment]
> I assume that this will happen for all object types?

this is the list of object types this warning is included in:
![image](https://user-images.githubusercontent.com/24838311/141965144-d61012cd-c7ad-4826-a5c2-1b95ba04001b.png)

**Comment by StefanMaron** [comment]
Next fix released. Report false warnings should also be gone

**Comment by jwikman** [comment]
Yes, `Report` works now.
But `XmlPort` and `Query` still creates warnings. :)

**Comment by StefanMaron** [comment]
Yes I just realized what you meant :D `Record` seems to be the exception as the definition is `table`

**Comment by StefanMaron** [comment]
> Yes, `Report` works now. But `XmlPort` and `Query` still creates warnings. :)

Should also be fixed now.

Thanks again for reporting 💪

**Comment by rvanbekkum** [comment]
With the latest update I got the following message:
```
Unable to load Analyzer assembly c:\Users\r.vanbekkum\.vscode\extensions\ms-dynamics-smb.al-8.1.540594\bin\Analyzers\BusinessCentral.LinterCop.dll : File c:\Users\r.vanbekkum\.vscode\extensions\ms-dynamics-smb.al-8.1.540594\bin\Analyzers\BusinessCentral.LinterCop.dll was not found.
```
Any idea what could be wrong?

**Comment by rvanbekkum** [comment]
The DLL is there. ;)

**Comment by StefanMaron** [comment]
maybe the download did not work. can you try to reload or try to start the download manually?

![image](https://user-images.githubusercontent.com/24838311/141968148-4d557a46-3aa2-4926-896c-2e06ab639fe6.png)

**Comment by rvanbekkum** [comment]
Yes, the download is successful, but it cannot load the DLLs.

**Comment by jwikman** [comment]
The update worked for me and XmlPort and Query is now OK 👍

**Comment by rvanbekkum** [comment]
The latest update also solved the load error for me.
